ballerina/tests/README.md

```diff
@@ -0,0 +1,16 @@
+Adding tests for get methods
+1. Albums
+2. Artist
+//3. AudioBooks {"audiobooks":[null,null,null]}
+4. Browse Categories
+//5. Chapters{"error": {"status": 404, "message": "Resource not found" } }
+//6. Episode
+7. Markets
+8. Playlist
+//9. Shows{"error": {"status": 404, "message": "Resource not found" } }
+10. Tracks
+11. User
+12. AlbumTracks
+13. ArtistId
+14. ArtistsAlbum
+15. Category
```

Comment:
@Sachindu-Nethmin we should follow the same tests README structure used by other connectors. (e.g. https://github.com/ballerina-platform/module-ballerinax-twitter/blob/main/ballerina/tests/README.md)